packages/bus-dereference/lib/ActorDereferenceParse.ts

```diff
@@ -5,21 +5,22 @@ import { passTestVoid } from '@comunica/core';
 import type { Readable } from 'readable-stream';
 import { PassThrough } from 'readable-stream';
 import type { IActionDereference, IActorDereferenceOutput, MediatorDereference } from './ActorDereference';
-import { ActorDereferenceBase, isHardError } from './ActorDereferenceBase';
+import { ActorDereferenceBase, isHardError, emptyReadable } from './ActorDereferenceBase';
 
 /**
  * Get the media type based on the extension of the given path,
  * which can be an URL or file path.
  * @param {string} path A path.
  * @param {Record<string, string>} mediaMappings A collection of mappings,
  * mapping file extensions to their corresponding media type.
- * @return {string} A media type or the empty string.
+ * @return {string} A media type or undefined.
  */
-export function getMediaTypeFromExtension(path: string, mediaMappings?: Record<string, string>): string {
+export function getMediaTypeFromExtension(path: string, mediaMappings?: Record<string, string>): string | undefined {
```

Comment:
Is this change really needed? The current code seems to work fine.

Reply:
It is not needed, but it makes it easier to distinguish between an actual value and not-a-value. I think this function is only used internally here, since nothing else broke when I changed it. :sweat_smile:

Reply:
I think it is used in the unit tests only, or something else, same with a lot of `public` attributes on classes in Comunica. But I will revert it!